AirCasting/Dashboard/EmptyMobileDashboardView.swift

```diff
@@ -49,11 +49,14 @@ private extension EmptyMobileDashboardViewMobile {
     
     private var emptyMobileDashboardText: some View {
         VStack(spacing: 14) {
-            Text(Strings.EmptyDashboardMobile.title)
-                .font(Fonts.boldTitle4)
-                .foregroundColor(Color.darkBlue)
-                .minimumScaleFactor(0.1)
-            
+            StringCustomizer.customizeString(Strings.EmptyDashboardMobile.title,
+                                             using: ["a"],
```

Comment:
will "a" be universal word here across all languages? If not it needs to be moved out